galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/cicsresource/ICicsBundle.java

```diff
@@ -186,5 +189,8 @@ public interface ICicsBundle {
 	 * Returns the CICS BUNDLE name as defined in the CICS Resource Definition
 	 * @return the CICS BUNDLE name
 	 */
-	public String getName();
+	public String getDefinitionName();

```

Comment:
Normally, you'd deprecate the public interface method, rather than renaming/removing it.

How sure are we that nobody is going to break as a result of this change ?
Could any of the Galasa users out there be using this interface ?

Reply:
Approved for building